### ISSUE TITLE
[request] Add helper to bundle multiple views

### ISSUE THREAD

**Issue by Undistraction** [kbparagua/paloma#30]
It would be great to be able to do something like:

```
ArticlesController.bundle(["edit", "update", "index"], function(){
  // Actions shared by all three
});
```
This could just be a helper method that does exactly the same thing as calling them individually.

**Comment by fahdoo** [comment]
+1

Is this being worked on?

**Comment by kbparagua** [comment]
@Undistraction @fahdoo Before callbacks are now implemented on `5.0.0`.
